homeassistant/components/motion_blinds/cover.py

```diff
@@ -146,6 +169,37 @@ def is_closed(self):
         """Return if the cover is closed or not."""
         return self._blind.position == 100
 
+    @callback
+    def push_callback(self):
```

Comment:
I believe this is only used by this class, so consider prefixing with "_"

Reply:
good sugestion, done